pkg/node/node.go

```diff
@@ -636,7 +636,7 @@ func NewBee(addr string, publicKey *ecdsa.PublicKey, signer crypto.Signer, netwo
 	}
 
 	kad, err := kademlia.New(swarmAddress, addressbook, hive, p2ps, pingPong, metricsDB, logger,
-		kademlia.Options{Bootnodes: bootnodes, BootnodeMode: o.BootnodeMode, StaticNodes: o.StaticNodes})
+		kademlia.Options{Bootnodes: bootnodes, BootnodeMode: o.BootnodeMode, StaticNodes: o.StaticNodes, IgnoreRadius: o.ChainEnable})
```

Comment:
isn't it the other way?
- if we have chain enabled we don't want to ignore the radius since the postage data is going to be there
- if we have the chain disabled then we want to ignore the radius

Reply:
yes, it seems I flipped it the other way around, fixing...